website/source/docs/acl/acl-rules.html.md.erb

```diff
@@ -561,3 +561,67 @@ session "admin" {
 Session rules are segmented by the node name they apply to. In the example above, the rules allow read-only
 access to sessions on node name with the empty prefix, allow creating sessions on the node named "app",
 and deny all access to any sessions on the "admin" node.
+
+#### Namespace Rules <span class="label-enterprise label-enterprise-lg">Enterprise</span>
+
+[Consul Enterprise](https://www.hashicorp.com/consul.html) 1.7.0 adds support for namespacing of many 
+Consul resources. ACL rules themselves can then be defined to only to apply to apply to specific namespaces.
+
+A Namespace specific rule would look like this:
+
+```hcl
+namespace_prefix "" {
+  
+  # grant service:read for all services in all namespaces
+  service_prefix "" {
+    policy = "read"
+  }
+  
+  # grant node:read for all nodes in all namespaces
+  node_prefix "" {
+    policy = "read"
+  }
+}
+
+namespace "foo" {
+  # grants permission to manage ACLs only for the foo namespace
+  acl = "write"
+  
+  # grants write permissions to the KV for namespace foo
+  key_prefix "" {
+    policy = "write"
+  }
+  
+  # grants write permissions for sessions for namespace foo
+  session_prefix "" {
+    policy = "write"
+  }
+  
+  # grants service:write for all services in the foo namespace
+  service_prefix "" {
+    policy = "write"
+  }
+  
+  # grants node:read for all nodes
+  node_prefix "" {
+    policy = "read"
+  }
+}
+```
+
+There are some restrictions to the rules that can be contained within a `namespace` or `namespace_prefix` rule that
+targets a non-`default` namespace.
+
+1. `operator` rules are not allowed.
+2. `event` rules are not allowed.
+3. `keyring` rules are not allowed.
+4. `query` rules are not allowed.
+5. `node` rules that attempt to grant `write` privileges are not allowed.
+
+In general all of the above are permissions that only an operator should have and thus granting these permissions can
+only be targeted at the default namespace.
+
+-> **Implicit namespacing:** Rules defined in a policy which itself resides in a namespace other than `default`, will 
```

Comment:
This is a complicated topic so its hard to explain. I re-wrote this section in an attempt to make it more clear. Let me know what you think:

"Rules and policies created within a namespace will inherit the namespace configuration. This means that rules and policies will be implicitly namespaced and do not need additional configuration. The restrictions outlined about will apply to these rules and policies. Additionally, rules and policies within a specific namespace are be prevented from accessing resources in another namespace."

Reply:
I updated with almost exactly this except I changed "are be" -> "are"